cypress/integration/CodeMirror.spec.js

```diff
@@ -1,7 +1,7 @@
-describe("CodeMirror", () => {
-  const languages = ["html", "javascript", "jsx", "css", "less", "vue"];
+import * as mocks from "../../sandpack-react/src/components/CodeEditor/languages-mocks";
```

Comment:
Refactor: take the advantage of some mocks and loop through them in order to generate some snapshots.